### DISCUSSION TITLE
第三方模块标准、模板讨论

### DISCUSSION THREAD

**Discussion by chexiongsheng** [Tencent/puerts#810] [Ideas]
faq里头关于source-map-support的使用，其实是不是能做成一个npm模块，比如叫“puerts-source-map-support”，执行npm -i puerts-source-map-support后，会安装依赖的source-map-support，以及unity环境下对fs、path的适配，用户自己可用，而不是faq上的文字化描述呢？

这是js模块的情况
还有C#模块，比如群上经常有人因为ILoader的实现，以及Filter配置等来咨询，是不是也可以做成各种各样的C#模块，仍然是npm install一下就可以用呢？

还有就是C#和js兼而有之的情况。

**Comment by throw-out** [comment]
建议用`@puerts/xxx`作为模块名, 文件结构如下, 看起来更友好
```
-node_modules
	-@puerts
		  +source-map-support
		  +......
	-......
```
